src/browser.qml

```diff
@@ -28,6 +28,9 @@ ApplicationWindow {
         if (!model || model.count === 0) {
             cover = Qt.resolvedUrl("cover/NoTabsCover.qml")
         } else {
+            if (cover != null && window.webView) {
+              window.webView.clearSurface();
```

Comment:
I think that better place for this is actually in the Overlay.qml.

Approximately at line 38 at the beginning of the else block:
```
            if (webView && webView.tabModel.count === 0) {
                webView.clearSurface();
            }
```
This way the all delays that are within the tab creation do not disturb.